knowledge_base/spark_jar_task/databricks.yml

```diff
@@ -0,0 +1,28 @@
+bundle:
+  name: spark-jar-task
+
+workspace:
+  host: https://e2-dogfood.staging.cloud.databricks.com
+  artifact_path: /Volumes/my/artifact/path
+
+artifacts:
+  my_java_code:
+    path: ./print_args
+    build: "javac PrintArgs.java && jar cvfm PrintArgs.jar META-INF/MANIFEST.MF PrintArgs.class"
```

Comment:
I get this is the smallest possible example, but I feel like we should make it a bit more real and call Maven to build a JAR here (and have a `pom.xml` file). Not blocking.